browser/css/jsdialogs.css

```diff
@@ -657,6 +658,12 @@ input[type='checkbox']:checked.autofilter, .jsdialog input[type='checkbox']:chec
 
 /* spinfield */
 
+.spinfieldcontainer {
+	margin-right: 5px;
```

Comment:
it should be not left / right, but start / end. This is important for RTL mode cc: @pedropintosilva

Reply:
https://github.com/CollaboraOnline/online/pull/8026